findagig-batch/src/test/java/findagig/batch/job/GigsWriterTest.java

```diff
@@ -0,0 +1,30 @@
+package findagig.batch.job;
+
+import findagig.batch.domain.entity.Gig;
+import findagig.batch.domain.factory.GigsFactory;
+import findagig.source.driver.SongKickDriver;
+import findagig.source.entity.Event;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static net.logstash.logback.argument.StructuredArguments.keyValue;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
+
+class GigsWriterTest {
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    void write() throws Exception {
+        GigsReader reader = new GigsReader();
```

Comment:
It's not testing the `GigsWriterTest`

Reply:
I was not sure if I should test a void method (and how to do it) .
Anyway, GigsWriterTest is now testing the write method from GigsWriter.